compose/cli/docker_client.py

```diff
@@ -32,4 +32,4 @@ def docker_client():
         )
 
     timeout = int(os.environ.get('DOCKER_CLIENT_TIMEOUT', 60))
-    return Client(base_url=base_url, tls=tls_config, version='1.18', timeout=timeout)
+    return Client(base_url=base_url, tls=tls_config, version='1.19', timeout=timeout)
```

Comment:
@aanand I've done just like you asked. I've renamed the env var to ```COMPOSE_API_VERSION``` and squashed to a single commit: [#1780](https://github.com/docker/compose/pull/1780/). Please don't hesitate to let me know if you need anything else.